protocols/virtualfund/virtualfund.go

```diff
@@ -678,9 +677,9 @@ type ObjectiveResponse struct {
 }
 
 // Response computes and returns the appropriate response from the request.
-func (r ObjectiveRequest) Response() ObjectiveResponse {
+func (r ObjectiveRequest) Response(myAddress types.Address) ObjectiveResponse {
```

Comment:
It seems a little awkward to have to pass this in, now -- I suppose that is a consequence of wanting to compute the channelId synchronously (i.e. as soon as the request is submitted, and before any store access or other async things happen)? Still, seems less bad than having to pass it across the API.